src/pages/layers/preview-map/index.js

```diff
@@ -1,23 +1,29 @@
+import { cx } from '@emotion/css';
 import { useGeometryStore, useLayersStore } from 'common/store';
 import Dropdown, { selectAllId } from 'components/dropdown';
 import { useFeatureFlags } from 'hooks';
 import { useEffect, useMemo, useRef, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import {
   dropdownContainer,
+  inlineSelectContainer,
   previewCanvas,
   previewContainerStyles,
+  previewDescription,
   previewDropdownStyles,
-  previewSelectContainer,
   previewSelectTitle,
   previewTitle,
+  previewTitleWrapper,
+  selectContainer,
 } from './index.style';
 import MapControls from './map-controls';
 import useTransformations from './useTransformations';
 
+const betaFeature = false;
+
 const geometrySelector = s => s.dwgLayers;
 const layersSelector = s => [s.dwgLayers, s.layers, s.getLayerNameError, s.previewSingleFeatureClass];
-const canvasSide = 500;
+const canvasSide = 516;
```

Comment:
it would be great if we could avoid this by layout but I understand it might not be possible here

Reply:
Yes. That is exactly what I am going to work on in the next update